src/scripts/index.js

```diff
@@ -0,0 +1,42 @@
+import "../style/style.css";
+
+(async () => {
+
+async function getSources(url) {
+  let response = await fetch(`${url}`);
+  return await response.json();
+}
+
+const apiKey = 'adb1c21889ad46df8aacb6d566c97770';
```

Comment:
In next task you can move such kind of data (constants, etc) into separate file like "config.js"